configure

```diff
@@ -17786,6 +17786,62 @@ $as_echo "no" >&6; }
 fi
 
 
+# yk integration
+ac_fn_c_check_header_mongrel "$LINENO" "yk.h" "ac_cv_header_yk_h" "$ac_includes_default"
+if test "x$ac_cv_header_yk_h" = xyes; then :
```

Comment:
Shouldn't we patch `configure.in` or (`configure.ac`)?

Reply:
Ohh! They imported their configure script into git! bizarre... Ignore me.

Reply:
CPython includes the generated `configure` in the repo. Notice that this PR contains the changes to `configure.ac` that generate this code.

Reply:
yeah, the penny only dropped when i scrolled down. I thought you had manually hacked the generated file.